_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -153,7 +153,7 @@ The `aria-label` [property][] is [global][] and thus [inherited][] for all [sema
 
 #### Passed Example 10
 
-This `button` element has an [explicit role][] of `none`. However, it is [focusable][] (by default). Thus it has a [semantic role][] of `button` due to [Presentational Roles Conflict Resolution][]. The `aria-pressed` [state][] is [supported][] for the `button` role.
+This `button` element has an [explicit role][] of `none`. However because it is [focusable][] (by default), it has a [semantic role][] of `button` due to [Presentational Roles Conflict Resolution][]. The `aria-pressed` [state][] is [supported][] for the `button` role.
```

Comment:
```suggestion
This `button` element has an [explicit role][] of `none`. However, because it is [focusable][] (by default), it has a [semantic role][] of `button` due to [Presentational Roles Conflict Resolution][]. The `aria-pressed` [state][] is [supported][] for the `button` role.
```